File: projects/store-or/src/main/java/com/robinfood/storeor/usecase/getconfigurationposbystore/IGetConfigurationPosByStoreUseCase.java
package com.robinfood.storeor.usecase.getconfigurationposbystore;

import com.robinfood.storeor.dtos.configurationposbystore.StorePosDTO;

import java.util.List;

public interface IGetConfigurationPosByStoreUseCase {

    List<StorePosDTO> invoke(Long storeId);
}

File: projects/posv2/app/src/main/kotlin/com/robinfood/app/usecases/getposrelatedtoastore/IGetConfigurationPosByStoreUseCase.kt
package com.robinfood.app.usecases.getposrelatedtoastore

import com.robinfood.core.dtos.posrelatedtoastore.StorePosDTO

/**
 * Use case List Configurations Pos by Store
 */
interface IGetConfigurationPosByStoreUseCase {

    /**
     * Sends a request to get List Configurations Pos by Store
     * @param storeId id of store
     * @param token token of authorization
     * @return List Configurations Pos by Store
     */
    suspend fun invoke(
        storeId: Long,
        token: String
    ): List<StorePosDTO>
}